src/rgw/rgw_user.cc

```diff
@@ -1706,7 +1706,10 @@ int RGWUser::execute_add(RGWUserAdminOpState& op_state, std::string *err_msg)
     if (!op_state.exclusive &&
         (user_email.empty() ||
 	 boost::iequals(user_email, old_info.user_email)) &&
-        old_info.display_name == display_name) {
+        old_info.display_name == display_name &&
+        op_state.will_modify_user_on_create()) {
+      cout << "unable to create user, user: " << op_state.user_id.to_str()
```

Comment:
This probably needs to be cerror not cout, since the result is parsable json, and we don't want to confuse the parser.